environment.rb

```diff
@@ -0,0 +1,139 @@
+module Environment
+    class Depot
```

Comment:
Indentation should be two spaces, be careful when setting the TAB key in your IDE, sometimes Git, just take a TAB as a 4-spaced-TAB ignoring the IDE spacing.